index.js

```diff
@@ -1,17 +1,60 @@
 class SortedList {
-  constructor() {}
+  constructor() {
+    this.items = [],
+    this.length = 0
+  }
 
-  add(item) {}
+  add(item) {
+    this.items.push(item);
+    this.items.sort(function(a, b){return a-b});
+    this.length =+1;
+  }
 
-  get(pos) {}
+  get(pos) {
+    if(this.items.includes(this.items[pos])){
+      return this.items[pos]
+    } else {
+      throw new Error('OutOfBounds')
+    }
+  }
 
-  max() {}
+  max() {
+    if (this.length>0) {
+      var max = this.items.reduce(function(a, b) {
```

Comment:
In this function, you are doing the same thing twice: your reduce is comparing and grabbing the max value, and you also return the max value with the Math method; you should stick to one only.